src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -1530,10 +1534,11 @@ export function makeScatterPlotData(
         const sampleCoverageInfo = coverageInformation[d.uniqueSampleKey];
         let profiledMutations = undefined;
         let profiledCna = undefined;
+        const noGenesInAxes = !horzData.hugoGeneSymbol && !vertData.hugoGeneSymbol;
         if (mutations || copyNumberAlterations) {
             const profiledReport = makeScatterPlotData_profiledReport(
-                horzData.hugoGeneSymbol,
-                vertData.hugoGeneSymbol,
+                (noGenesInAxes && selectedGeneForClinicalAttrVsTreatment) ? selectedGeneForClinicalAttrVsTreatment.hugoGeneSymbol : horzData.hugoGeneSymbol,
```

Comment:
isn't it redundant to check if no genes in axis? isn't it true that theres definitely no genes in axis if you're passing in a gene for clinical attr vs treatment plot?

Reply:
Last time I checked it's true that for clinical vs treatment plot, there are no genes in both axes. I agree that it's a bit redundant, perhaps checking if `selectedGeneForClinicalAttrVsTreatment` exists is good enough because this parameter is passed as an argument only in this specific case. I will remove the `noGenesInAxes` condition.